src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The Gateway engine Pod has been renamed from `submariner` to `submariner-gateway`.
+* The helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the operator instead of `subctl`.
+* Each Submariner Pod now has it's own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric was renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy Service Discovery
+  only via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* `lighthouse-coredns` no longer relies on `Gateway` object to determine `localClusterID`.
+* Lighthouse uses `coredns-custom` configmap for AKS deployments.
```

Comment:
This doesn't seem user-facing either or, if it is, I think it needs more explanation (I'm not clear on it). @vthapar

Reply:
Turns out the configmap name is not constant but varies from deployment to deployment. So good to capture this in case it is not working for someone's AKS deployment.